include/deal.II/numerics/data_out_stack.h

```diff
@@ -304,7 +314,8 @@ class DataOutStack : public DataOutInterface<dim+1>
   DeclException1 (ExcInvalidNumberOfSubdivisions,
```

Comment:
This and some of the other exceptions are duplicated exactly. I vote for either moving them to a generic header or declaring them once and using a typedef in the other places.